tests/test_web/test_api_v1.py

```diff
@@ -1,5 +1,6 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 from unittest import mock
+import json
```

Comment:
```suggestion
import json
from unittest import mock
```

Reply:
Oh `import` comes first? I always thought it was the other way around.

Reply:
We just keep it alphabetized. We don't have a convention of putting `from` or `import` in a separate order.